docs/UserGuide.md

```diff
@@ -864,6 +912,19 @@ delivery? No worries! Just edit the details of a delivery simply by using this f
 **Format:** `delivery edit DELIVERY_ID [--name DELIVERY_NAME] [--customer CUSTOMER_ID] [--date EXPECTED_DELIVERY_DATE]
 [--status STATUS] [--note NOTE]`
 
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
+
+**Note:**
+
+* At least one of the optional fields must be provided.
+* `DELIVERY_ID`{.swfit} must be an integer greater than 0.
```

Comment:
`DELIVERY_ID`{.swift} must be an integer greater than 0 that corresponds to an existing Delivery in the address book.